starknet_devnet/__init__.py

```diff
@@ -1,5 +1,23 @@
 """
 Contains the server implementation and its utility classes and functions.
 """
+import sys
+from starkware.crypto.signature.fast_pedersen_hash import pedersen_hash
+from crypto_cpp_py.cpp_bindings import cpp_hash
+
 
 __version__ = "0.2.2"
+
+
+def patched_pedersen_hash(x: int, y: int) -> int:
```

Comment:
This method doesn't comply with the project's linting standard. See yourself by running `poetry run pylint starknet_devnet test`

Reply:
Oh, thanks for pointing this out :+1: Changed in latest patchset.